src/main/java/com/example/b2b_opportunities/Dto/Request/ExperienceRequestDto.java

```diff
@@ -9,10 +9,6 @@
 @Setter
 public class ExperienceRequestDto {
     @Min(0)
-    @Max(11)
+    @Max(Integer.MAX_VALUE)
```

Comment:
I would go with a much lower number. 600 is more than enough and I would consider everything above it as a mistake.